CMakeLists.txt

```diff
@@ -63,6 +63,8 @@ option(IPDK_ROLE_FIX "Enable IPDK role config changes" ON)
 option(SET_RPATH    "Set RPATH in libraries and executables" OFF)
 option(WITH_KRNLMON "Enable Kernel Monitor support" ON)
 option(WITH_OVSP4RT "Enable OVS support" ON)
+option(LNW_V2 "Linux Networking version V2" OFF)
```

Comment:
Please remove these options. I've modified the LNW_VERSION implementation to define a cmake variable corresponding the variable that is passed to the C/C++ compiler.